### DISCUSSION TITLE
Inventory Reports being discontinued

### DISCUSSION THREAD

**Discussion by nrogers805** [amzn/selling-partner-api-models#3084] [General]
I have had notifications on my seller account that the following reports will be discontinued at the end January 2023

– Monthly Inventory History
– Inventory Event Detail
– Inventory Adjustments
– Inventory Reconciliation
– Received Inventory

Does this also apply to the equivalent reports in SP-API ? I have not seen any SP-API specific announcements about this so just wanted to check.

**Comment by ericahrowe** [comment]
According to our communication with Amazon, this does apply to the SP-API. I can't find a public announcement for it, but Amazon has provided recent email communication about the changes.

**Reply by nrogers805** [reply]
> According to our communication with Amazon, this does apply to the SP-API. I can't find a public announcement for it, but Amazon has provided recent email communication about the changes.

The new ledger report does not contain the level of detail that the adjustment report contains - how do we make a complaint ?

**Reply by sungolivia** [reply]
https://developer.amazonservices.com/support
